guides/common/modules/proc_creating-compute-profiles.adoc

```diff
@@ -8,6 +8,12 @@ A default installation of {ProjectName} contains three predefined profiles:
 * `2-Medium`
 * `3-Large`
 
+You can apply the compute profiles to the following compute resource providers:
+
+* VMware
+* Libvirt
+* Red Hat Virtualization (deprecated)
```

Comment:
```suggestion
* {oVirt}
```

If you really want to say "deprecated", I'd suggest to use "if satellite" and "if not satellite". See https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-base.adoc#L76 and https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-satellite.adoc#L76

Reply:
Or maybe add "depreacted" to the attribute?